stdlib/testing/kapacitor/combine_pivot_test.flux

```diff
@@ -35,21 +35,16 @@ outData = "
 ,,0,memory,2018-12-19T22:14:10Z,6,9
 ,,0,memory,2018-12-19T22:14:20Z,3,8
 "
+t_combine_join = (table=<-) => table
+    |> range(start: 2018-12-15T00:00:00Z)
+    |> drop(columns: ["_start", "_stop"])
+    |> pivot(rowKey: ["_time", "_measurement"], columnKey: ["_field"], valueColumn: "_value")
 
-t_combine_join = (table=<-) =>
-    (table
-        |> range(start: 2018-12-15T00:00:00Z)
-        |> drop(columns: ["_start", "_stop"])
-		|> pivot(rowKey:["_time", "_measurement"], columnKey: ["_field"], valueColumn: "_value"))
-
-test _combine_join = () =>
-	({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_combine_join})
-
-
+test _combine_join = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_combine_join})
 // Equivalent TICKscript query:
 //
 // stream
 //    |from()
 //      .measurement('request_latency')
 //    |combine(lambda: "_field"=='user1;, lambda: "_field"=='user2')
-//      .as('user1', 'user2')
\ No newline at end of file
+//      .as('user1', 'user2')
```

Comment:
Whether a project/language/file has newlines at the end of files is one of the greatest annoyances of my entire career, so I'm glad to see our formatter being opinionated here.